_includes/tutorials/fk-joins/kstreams/code/tutorial-steps/dev/track-purchases.json

```diff
@@ -0,0 +1,7 @@
+100:{"id": 100, "album_id": 5, "song_title": "Houses Of The Holy", "price": 0.99}
```

Comment:
This is the correct format, it allows for passing in keys to `kafka-avro-console-producer`